src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,10 +3,12 @@
 import java.util.Random;
 
 public class Lottery {
+    private static final Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+
     public Ball getRandomBall() {
-        ColorSupplier supplier = new ColorSupplier();
         String color = supplier.getRandomColor();
-        int number = new Random().nextInt(100);
+        int number = random.nextInt(100);

```

Comment:
magic number should be a constant
Also, a bound parameter of `nextInt()` is exclusive, so we should account for that, check Javadoc of the nextInt() for details